### ISSUE TITLE
[bug]: Browser title does not get updated on accessing categories. Updates only after manual browser reload.

### ISSUE THREAD

**Issue by dpatil-magento** [magento-research/pwa-studio#1289]
**Describe the bug**
Browser or Window title does not get updated on accessing categories. It gets update if user is on category and does browser reload.

**To Reproduce**
1. Navigate to Venia home page.
2. Access any category.

**Expected behavior**
Browser title should change to category name.

**Screenshots**
In this screen Browser title still says Home page - Venia  instead of Tops - Venia
![image.png](https://images.zenhubusercontent.com/5c585c5aecb0076f3e898431/5b44f1ba-8e94-4cf3-a3b0-72e1213a33df)

**Comment by tjwiebell** [comment]
Duplicate of #422, which also includes meta tags in the bug scope.
